app/javascript/article-form/components/LoadingPreview.jsx

```diff
@@ -0,0 +1,38 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+export const LoadingPreview = ({ version }) => {
```

Comment:
I think feel free to add new CSS selectors if need be 👍 We have e.g. `crayons-story__body` for the article loading, but maybe we could create something like `crayons-preview__cover-image`, etc. It could go in the `article-form.scss`

Reply:
Based on your comment I have applied following changes:
1. In `acticle-form.scss` created `crayons-preview__cover` which uses `--body-bg` as background color.
2. For all other styles in LoadingPreview I have used the existing selectors like `crayons-scaffold`, `crayons-story__body`, `crayons-story__indention` ,etc.
3. I have add some new width values in `_generator.scss` which are useful in creating the UI.

I have done these changes as it was the easiest working solution for me but if there are any changes needed in these do let me know.